db/utils.ts

```diff
@@ -1,7 +1,39 @@
-import { drizzle } from 'drizzle-orm/postgres-js';
+import { neon } from '@neondatabase/serverless';
+import {
+  drizzle as drizzleNeon,
+  type NeonHttpDatabase,
+} from 'drizzle-orm/neon-http';
+import {
+  drizzle as drizzlePostgres,
+  type PostgresJsDatabase,
+} from 'drizzle-orm/postgres-js';
 import postgres from 'postgres';
 
-export function createDbConnection(databaseUrl: string) {
+export type Database = PostgresJsDatabase | NeonHttpDatabase;
+
+export function getDeletedRowCount(result: any): number {
```

Comment:
Can we improve type safety here? E.g. union of the postgres result / neon result / whatever else we need as we need it?